src/app/api/(auth)/forgot-password/route.ts

```
@@ -0,0 +1,41 @@
+    const { error } = await supabase.auth.resetPasswordForEmail(email, {
+      redirectTo: `${process.env.NEXT_PUBLIC_SITE_URL}/reset-password`,
+    });
+
+    if (error) {
+      // Return any Supabase-related error
+      return NextResponse.json({ error: error.message }, { status: 500 });
+    }
+
+    // Return success response
+    return NextResponse.json({ message: "Password reset email sent" }, { status: 200 });
+  } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)